landoapi/models/treestatus.py

```diff
@@ -198,14 +174,7 @@ class StatusChange(Base):
     reason = db.Column(db.Text, nullable=False)
 
     # The status the trees were changed to.
-    status = db.Column(
-        db.Enum(
-            TreeStatus,
-            # Use the values of the enum in the DB instead of the keys.
-            values_callable=lambda x: [i.value for i in x],
-        ),
-        nullable=False,
-    )
+    status = db.Column(db.Enum(TreeStatus), nullable=False)
```

Comment:
Does this need a default?

Reply:
No, a missing status here is a programming error.